test/e2e/kubectl/kubectl.go

```diff
@@ -472,8 +470,12 @@ var _ = SIGDescribe("Kubectl client", func() {
 				framework.Failf("--host variable must be set to the full URI to the api server on e2e run.")
 			}
 
-			ginkgo.By("Starting goproxy")
-			testSrv, proxyLogs := startLocalProxy()
+			ginkgo.By("Starting http_proxy")
+			var proxyLogs bytes.Buffer
+			testSrv := httptest.NewServer(utilnet.NewHTTPProxyHandler(func(req *http.Request) bool {
```

Comment:
do we need this new library here or shouldn't be enough with `NewSingleHostReverseProxy` only?